docs/metrics/icd11foundation.md

```diff
@@ -2,15 +2,15 @@
 
 **IRI:** http://purl.obolibrary.org/obo/mondo-ingest/components/icd11foundation.owl
 
-**Version IRI:** http://purl.obolibrary.org/obo/mondo-ingest/releases/2024-02-17/components/icd11foundation.owl
+**Version IRI:** http://purl.obolibrary.org/obo/mondo-ingest/releases/2024-03-22/components/icd11foundation.owl
```

Comment:
## FYI: Size of `icd11foundation-unmapped.owl`
I got:
```
remote: warning: File src/ontology/unmapped/icd11foundation-unmapped.owl is 70.64 MB; this is larger than GitHub's recommended maximum file size of 50.00 MB
```

The limit is 100mb, and this is getting a little close. I take it this will often be the case for `*-unmapped.owl` files for new sources with lots of terms.

Reply:
I agree. My personal feeling is to commit it anyways, but if you or @twhetzel feel like this should be gitignored, its also fine. No issue either way